server/src/main/java/org/apache/iotdb/db/engine/storagegroup/TsFileProcessor.java

```diff
@@ -1148,11 +1151,11 @@ public void flushOneMemTable() {
         }
       } catch (Exception e) {
         logger.error(
-            "{} meet error when flush FileMetadata to {}, change system mode to read-only",
+            "{}: {} marking or ending file meet error",
             storageGroupName,
-            tsFileResource.getTsFile().getAbsolutePath(),
+            tsFileResource.getTsFile().getName(),
```

Comment:
```suggestion
            tsFileResource.getTsFile().getAbsolutePath(),
```
Use absolute path is better for debugging

Reply:
Fixed.